<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IVHBM7A" failures="1" tests="12" name="Test" time="247.95" errors="0" timestamp="2025-04-02T15:10:52 IST">
  <testcase classname="api.test.UserTest2" name="TestPostUser" time="14.385"/>
  <testcase classname="api.test.UserTest2" name="TestGetUser" time="12.484"/>
  <testcase classname="api.test.UserTest2" name="TestUpdateUser" time="43.016"/>
  <testcase classname="api.test.UserTest2" name="TestDeleteUser" time="7.121"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="13.114"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="8.874"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="8.088"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="9.09"/>
  <testcase classname="api.test.DDTests" name="DeleteUSer" time="64.04">
    <failure type="org.awaitility.core.ConditionTimeoutException" message="api.test.DDTests$$Lambda/0x0000027c105d7268 expected the predicate to return &amp;lt;true&amp;gt; but was null within 1 minutes.">
      <![CDATA[org.awaitility.core.ConditionTimeoutException: api.test.DDTests$$Lambda/0x0000027c105d7268 expected the predicate to return <true> but was null within 1 minutes.
at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
at org.awaitility.core.AbstractHamcrestCondition.await(AbstractHamcrestCondition.java:86)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:691)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:708)
at api.utilities.ApiUtils.retryApiCall(ApiUtils.java:16)
at api.test.DDTests.DeleteUSer(DDTests.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
... 36 more
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- DeleteUSer -->
  <testcase classname="api.test.DDTests" name="DeleteUSer" time="23.558"/>
  <testcase classname="api.test.DDTests" name="DeleteUSer" time="17.131"/>
  <testcase classname="api.test.DDTests" name="DeleteUSer" time="18.017"/>
</testsuite> <!-- Test -->
